### ISSUE TITLE
Contact product owner about behaviour while pending acceptance

### ISSUE THREAD

**Issue by PaoVal** [CMPUT301F21T23/HabitTracker#53]
This issue is related to user story #48 
Might want to contact product owner and ask questions about what SHOULD happen while the sender is waiting on a response.

What should the sender see?
Should there be a place where the sender is able to see pending requests?

Should the user be notified when a request has been accepted?
Should the user be notified when a request has been denied?

**Comment by zarifmahfuz** [comment]
We know the answers to the questions asked.

 What should the sender see? Information about the status of the request.

Should there be a place where the sender is able to see pending requests? Yes, in the user's profile.

Should the user be notified when a request has been accepted? No, not part of the requirements.

Should the user be notified when a request has been denied? No, not part of the requirements.
